sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample5_SignVerify.md

```diff
@@ -20,21 +20,21 @@ First, we'll create both an RSA key and an EC key which will be used to sign and
 
 ```C# Snippet:KeysSample5CreateKey
 string rsaKeyName = $"CloudRsaKey-{Guid.NewGuid()}";
-var rsaKey = new CreateRsaKeyOptions(rsaKeyName, hardwareProtected: false)
+var rsaKeyOptions = new CreateRsaKeyOptions(rsaKeyName, hardwareProtected: false)
 {
     KeySize = 2048,
 };
 
 string ecKeyName = $"CloudEcKey-{Guid.NewGuid()}";
-var ecKey = new CreateEcKeyOptions(ecKeyName, hardwareProtected: false)
+var ecKeyOptions = new CreateEcKeyOptions(ecKeyName, hardwareProtected: false)
 {
     CurveName = KeyCurveName.P256K,
 };
 
-KeyVaultKey cloudRsaKey = keyClient.CreateRsaKey(rsaKey);
+KeyVaultKey cloudRsaKey = keyClient.CreateRsaKey(rsaKeyOptions);
```

Comment:
Since you're disambiguating, you might also drop "cloud" as a prefix to this local and `cloudEcKey` below.

Reply:
Wouldn't the "cloud" prefix still be good, since the actual keypair was created in the Azure cloud, and I get a reference to a representation of this cloud-created object?

Reply:
Changed it now, and even removed the "cloud" prefixes. However, when I run the `Update-Snippets.ps1`, none of the md files have changed according to `git status`; I would have expected the prefix-change to be reflected as well?

Reply:
They should've, yes. Did you get any errors? Did you pass `-ServiceDirectory keyvault`? Though, even if you didn't do the latter, that would just run over the entirety of the _sdk_ folder.

For expediency and because we appreciate your contribution, I can make the updates inline and get this PR merged.

Reply:
@heaths Yes I used exactly the command as you said, and it did not throw any errors. It said "Replaced <file>" for all md files, including the Sample5_SignVerify.md. I saw the timestamp was touched but it the contents were unchanged. I ran the PS script on ExecutionPolicy Process Unrestricted. Does PowerShell also need an additional permission to allow file content changes (not that I ever had troubles when processing and transforming files with PS scripts)?